translations/fr.po

```diff
@@ -785,15 +784,17 @@ msgstr ""
 "  - multiple search terms are allowed\n"
 "\n"
 "- **Selecting which exercises can be seen by the students**\n"
-"  - select the exercises with a click (the box on top (de)selects all; you can\n"
+"  - select the exercises with a click (the box on top (de)selects all; you "
```

Comment:
Do you understand why a newline has been inserted here? Did you do this change?

Reply:
Dear @yurug, yes, I noticed that as well after doing `make update-fr-translation`.
Note: this is not a true newline (there is no `\n`) but just an automatic wrapping triggered by gettext.
So I think we can just keep the new version as is

Reply:
Probably a difference in GNU gettext version ? The new line cuts aren't very pretty, but not a real problem either.